lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -188,53 +220,62 @@ func (plmgr *PluginManager) run() error {
     for {
```

Comment:
This is the only goroutine which is expected to run continuously. At system shutdown all the other goroutines are expected to exit.  This will prevent  the plugin manager to exit prematurely.

Reply:
OPtimize run()
club ...
handle  in the request()  itself ...

Reply:
TODO